src/S3/MultipartCopy.php

```diff
@@ -128,15 +129,17 @@ private function createPart($partNumber, $partsCount)
             $data[$k] = $v;
         }
 
-        list($bucket, $key) = explode('/', ltrim($this->source, '/'), 2);
-        $data['CopySource'] = '/' . $bucket . '/' . implode(
```

Comment:
This implode sequence is lost in the new `else`.

Reply:
It looks like this has been added back. However, I'm worried this might have slipped through - are there tests (better yet, integration tests) for this to update or could some be added to cover this class and it's edges?